sys/net/link_layer/ieee802154/submac.c

```diff
@@ -300,8 +301,8 @@ int ieee802154_submac_init(ieee802154_submac_t *submac)
     ieee802154_radio_request_on(dev);
 
     /* generate EUI-64 and short address */
-    luid_get_eui64(&submac->ext_addr);
-    luid_get_short(&submac->short_addr);
```

Comment:
You can also drop the `luid` dependency now.

Reply:
true!

Reply:
done!